### ISSUE TITLE
[Accessibility][Related Products] Product images do not have a mechanism to calculate an accessible name (missing alt attribute)

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1770]
**Steps to Reproduce**
1. With NVDA running, navigate to product images.

**Current Behavior**
Product images do not have an alt attribute and are announced by NVDA with their file path instead.

**Expected behavior/code**
All meaningful images have informative, concise alternative text using the alt attribute.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Screen reader users

**WCAG SC**
1.1.1 Non-Text Content

**Code snippet**
```
<img data-gallery-role="currentimage" src="/content/dam/core-components-examples/library/cif-sample-assets/catalog/product/bolo-sport-watch/wg01-bk-0.jpg" class="product__image">
```

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by mhaack** [comment]
Fixed by CIF-2279 via https://github.com/adobe/aem-core-cif-components/pull/666
